### ISSUE TITLE
my settings reset (v8.0.29) & YouTube videos won't download (v8.0.26)

### ISSUE THREAD

**Issue by QuestioningEspecialy** [subhra74/xdm#1128]
**Describe the bug**
My settings reset after installing v8.0.29 and YouTube videos stopped downloading suddenly on v8.0.26.

**To Reproduce**
1. Download & Install v[8.0.26](https://github.com/subhra74/xdm/releases/tag/8.0.26)'s Windows 10 msi file
2. Use as normal to download files and YouTube videos
3. Notice yesterday that YouTube videos stopped downloading ("Invalid response from server")
4. Download & Install v[8.0.29](https://github.com/subhra74/xdm/releases/tag/8.0.29)'s Windows 10 msi file
5. Notice the same download error
6. Startup PC this morning
7. Notice XDM already launched in the background
8. Open XDM
9. Notice the first-time launch popup telling things I glossed over this time
10. Open Settings
11. Pretty sure the Download categories' Download folder reset to initial defaults, but unsure
12. Notice the "Launch XDM when system starts" box was rechecked
13. Notice downloading setup files with XDM still works

**Expected behavior**
Instead of the "Invalid response from server" popup, the download should just start.
Instead of Settings being reset, everythng should've stayed as it was on the previous version.

**Screenshots**
![image](https://github.com/subhra74/xdm/assets/115519265/e114a468-4d63-43f6-affc-c30401115c01)

**please complete the following information:**
 - OS: Windows 10
 - Browser: Waterfox (XDM extension not installed)
 - XDM addon Version: N/A
 - XDM Version: 8.0.26 & 8.0.29

**Log Generation**
 - https://github.com/subhra74/xdm/wiki/Generate-log-for-troubleshooting

This process isn't working for me. Neither "C:\Program Files (x86)\XDM" nor "C:\Programs Files\XDM" even though the installation folder's the former. 🤷🏿‍♂️
I get the following response in cmd whether it's ran as administrator or not:

> 'C:\Program' os not recognized as an internal or external command, operable program or batch file.

**Additional context**
The non-beta version(s) worked for me. According to my old setup files, I started with "xdmsetup-8.0.18-x86.msi". Not sure where I got it from, though, but 'eh.

**Comment by nickr61** [comment]
yep Youtube downloads no longer work for me either, same , invalid response from server. I expect youtube have tightened the code to stop downloading
